scripts/e2e-test.sh

```diff
@@ -65,23 +69,66 @@ if [ -n "$REGISTRY" ]; then
   export e2e_docker_registry="$REGISTRY"
 fi
 
-test_failed=
+test_failed=0
+
+function cleanup {
+    # Workaround for tests which leave dangling resources,
+    # usually these will be 3rd party tests.
+    # In some cases a future
+    # feature/bug-fix will remove the need for
+    # cleanup external to the test.
+    #
+    # Affected tests:
+    #   - csi (see CAS-566) - delete leftover MSVs
+    if [[ -x ./cleanup.sh ]]; then
+        ./cleanup.sh
```

Comment:
this looks weird. Basically you are trying to call test suite specific cleanup written in shell script if the file exists instead of cleaning up the resources in the test suite itself. That does not seem right. Or could you explain why the test suite does not do the cleanup?

Reply:
Agreed it looks odd. But these are 3rd party tests and I am reluctant to make more changes.
The easiest way to fix the MSV leakage was a shell script, until CAS-566 is fixed.